apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts

```diff
@@ -177,6 +177,22 @@ export const StyledMenuItem = styled(Menu.Item)`
     background-color: ${colors.defaultWhite[5]};
     font-weight: 700;
   }
+  > span {
```

Comment:
Please, move this styles to apps/condo/domains/user/components/UserMenu.tsx  
and this is very dangerous to declare such styles without & prefix
are you sure you do not kill styles somewhere else?

Reply:
Why dangerous? AFAIK, `> span` means `span` as a direct descendant.

Reply:
```
.ant-space-item:nth-child(2) {
    width: 100%;
  }
.ant-space-item:nth-child(2) {
    overflow: hidden;
}
.ant-space-item:nth-child(2) > div {
    overflow: hidden;
    text-overflow: ellipsis;
}
```